doc/releases/migration-guide-4.0.rst

```diff
@@ -278,6 +278,16 @@ Bluetooth Audio
   :c:func:`bt_bap_unicast_server_unregister` has been called.
   (:github:`76632`)
 
+* The Broadcast Audio Scan Service (BASS) can now be registered and unregistered dynamically
+  at runtime within the scan delegator. Two new APIs, :c:func:`bass_scan_delegator_register()`
+  and :c:func:`bass_scan_delegator_unregister()`, have been introduced to manage both BASS and
+  scan delegator registration and initialization dynamically.
+  This change allows more flexibility when registering or unregistering scan delegator and BASS
+  related functionality (e.g., Auracast) without requiring build-time configuration. Existing
+  applications that statically register BASS may need to be updated to use these new APIs if
+  dynamic control is required.
```

Comment:
```suggestion
  related functionality (e.g., Auracast) without requiring build-time configuration. Existing
  need to be updated to use these new APIs.
```

We should not have both dynamic and static registration.

Reply:
True, corrected.